.circleci/config.yml

```diff
@@ -19,97 +19,83 @@ jobs:
     steps:
       - checkout
       - run: find . -name '*.bash' -exec shellcheck {} \;
-  llvm-600-debug:
+  ubuntu-llvm-600-debug:
```

Comment:
this is incorrect here and elsewhere.

there is no ubuntu-llvm-600-debug etc.

Reply:
See first reply [above](https://github.com/ponylang/ponyc/pull/2663#discussion_r191079706)